### ISSUE TITLE
default panel size setting does not agree with actual layout

### ISSUE THREAD

**Issue by gordonwoodhull** [att/rcloud#2143]
The first time a user encounters RCloud 1.6, the Settings show "Arrange panels by size" enabled. But it is actually not enabled. One must toggle the setting in order to see it take effect.

You can delete the setting and simulate an initial login by running

```r
rcs.rm(usr.key(notebook='system', 'config', 'panel-layout-by-size'))
```

It may be that the settings_frame.js doesn't read a default value of `true` correctly.

**Comment by shaneporter** [comment]
Thanks for the repo instructions @gordonwoodhull

**Comment by shaneporter** [comment]
Resolved in PR #2145.
